doc/modules/array_api.rst

```diff
@@ -199,17 +200,15 @@ it supports the Array API. This will enable dedicated checks as part of the
 common tests to verify that the estimators' results are the same when using
 vanilla NumPy and Array API inputs.
 
-To run these checks you need to install
-`array_api_compat <https://github.com/data-apis/array-api-compat>`_ in your
-test environment. To run the full set of checks you need to install both
+To run the full set of checks you need to install both
 `PyTorch <https://pytorch.org/>`_ and `CuPy <https://cupy.dev/>`_ and have
 a GPU. Checks that can not be executed or have missing dependencies will be
```

Comment:
This is half suggestion and half request for comment (ill make a new PR):

```suggestion
To run the full set of checks you need to install 
`array-api-strict <https://data-apis.org/array-api-strict/>`_ which does not require a GPU. If you
have a GPU you can install `PyTorch <https://pytorch.org/>`_ or `CuPy <https://cupy.dev/>`_ instead.
Checks that can not be executed or have missing dependencies will be
```

I should have updated the docs in #30090

Reply:
Let's discuss/wordsmith in a new PR. At least for me this is not important enough to delay merging this.

I'll make a PR